src/pages/workspaces/workspace/Data.js

```diff
@@ -447,6 +447,18 @@ const WorkspaceData = _.flow(
 
   const toSortedPairs = _.flow(_.toPairs, _.sortBy(_.first))
 
+  const getRunningImportJobs = async () => {
+    try {
+      const runningJobs = await Ajax(signal).Workspaces.workspace(namespace, name).listImportJobs(true)
+      _.map(job => {
+        const jobStatus = _.lowerCase(job.status)
+        if (!(jobStatus === "success" || jobStatus === "error" || jobStatus === "done"))
+          asyncImportJobStore.update(Utils.append({ targetWorkspace: { namespace, name }, jobId: job.jobId }))
+      }, runningJobs)
+    } catch (error) {
+      reportError('Error loading running import jobs in this workspace')
+    }
+  }
```

Comment:
This change means that the Data tab checks for running import jobs and polls any that exist. The user impact is that reloading the data page will indicate running jobs.

Reply:
should this pop open the "import in progress" modal as well?

I worry a little bit here about cases where a workspace has a failed import that's stuck in some running state … in that condition, we'll *always* show a running job. I realize the proper fix for this is in import service/Rawls, to make sure we never have stuck jobs!

Reply:
We could - I was thinking, in the case that a job got stuck in progress, a modal popping up might be annoying for the user. 

Yeah, I was a little worried about stuck imports too. I think one way to fix it might be to pass up when the job started, and the UI could ignore jobs that have been running for more than, say, 24 hours. (obviously, that's more of a quick, in-case-things-go-wrong solution, ideally, as you said, the back end should be able to handle stuck jobs)

Reply:
all sounds good - I say leave this UI PR as-is, and if/when we realize stuck jobs are a problem in practice, we deal with it in the back end